### PR TITLE
Remove some more globals from test.

### PR DESCRIPTION
**Pull request by jasongilanfarr**

### CODE REVIEW

File: src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala [hunk 1]
```diff
@@ -455,7 +455,8 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
 
   test("Create a new app (that uses secrets) successfully") {
     Given("The secrets feature is enabled")
-    AllConf.withTestConfig(Seq("--enable_features", "secrets"))
+    configArgs = Seq("--enable_features", "secrets")
+    resetAppsResource
```
Comment (jeschkies) [inline src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala]: Hm, does `configArgs` have to be mutable? It's probably too much for this diff but it feels that `configArgs`, `appsResource` etc are fixtures more or less unique to each test case.

Comment (jasongilanfarr) [inline src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala]: Almost everything in this class is mutable.

Comment (jasongilanfarr) [inline src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala]: Refactoring it into a fixture wasn't worth the effort.

Comment (jeschkies) [inline src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala]: Kind of hat I meant. It's probably too much effort to refactor this.

**Comment by jdef** [comment]
:+1: :+1: :+1:

**Pull request merged**